challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+it is an absolute path since it starts with "/" and it starts from the root.
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+ we should use an absolute path since we need the full url to get the font file, and it is outside of our working directory.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
- 
+ we should use relative path since the image is within our working directory.
```

Comment:
Well, not quite. It's less about which directory we're working in, and more which _project_ the file is in. We want to use Relative paths on our links within a project so that if we should move the project folder, our links will not break.